input/pagecontent/search-parameters.md

```diff
@@ -12,4 +12,12 @@ NOTE: The following search parameters **SHALL NOT** be interpreted on their own
 ### Patient
 
 - [indigenous-status](SearchParameter-indigenous-status.html)
-- [patient-gender-identity](SearchParameter-patient-gender-identity.html)
```

Comment:
Missing the other resources the new search parameter applies to incl Person and PractitionerRole.

Reply:
Added Person and PractitionerRole to the list